community/sig-spec/README.md

```diff
@@ -0,0 +1,24 @@
+# SIG Spec
+
+A Special Interest Group (SIG) concerning the official SPIFFE specifications.
+
+### Meetings:
+* Date/Time: TBD
+* [Meetings Notes](https://docs.google.com/document/d/1p2BbRWN_7z6nkBMj-h1mAJAJxxKqNeFiV4IplZ_wU4c)
+* Calendar ICS (TBD) (See this [Google support page](https://support.google.com/calendar/answer/37100?co=GENIE.Platform%3DDesktop&hl=en) to learn more about importing .ics files)
+
+### Contact:
+* [Slack Channel](https://spiffe.slack.com/messages/sig-spec/)
+* [Google Group](https://groups.google.com/a/spiffe.io/d/forum/sig-specification)
+
+### Goals:
+* Preside over all [SPIFFE specifications](https://github.com/spiffe/spiffe#spiffe-standards)
+	* Ensure that Pull Requests and Issues are addressed
+	* Make recommendations for changes and updates
+	* Ensure wide interoperability
```

Comment:
Note: this includes owning creation and maintenance of conformance-style test suites.

Reply:
Thank you, fixed